src/index.html

```diff
@@ -10,13 +10,29 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo header!!</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="img" href="index.html"><img class="img" src="images/logo.png" alt="logo"></a>
```

Comment:
The `img` class is used both for the anchor tag and the image tag. This can lead to styling conflicts if different styles are intended for these elements. Consider using distinct class names for clarity and to avoid potential CSS conflicts.